app/src/pages/SingleContent.vue

```diff
@@ -21,10 +18,20 @@ const props = defineProps<Props>();
 const content = db.getBySlugAsRef<ContentDto>(props.slug);
 const tagsContent = ref<ContentDto[]>([]);
 
+const isExpiredOrScheduled = computed(() => {
+    if (!content.value) return false;
+    return content.value.publishDate! > Date.now() || content.value.expiryDate! < Date.now();
```

Comment:
I was triggered by the use of `!` here - generally it's good to be super careful when using it in production code (I care a lot less about it in test code). Are you certain there is always a `publishDate` AND `expiryDate` here? Both are optional. I can guess that a publish date is usually available for posts in the app, but expiry dates will probably not be present for the vast majority of posts. And even a publish date is not guaranteed, perhaps we add some way to view draft posts in the future?

So I think we need to do extra checks here (`content.value.publishDate && content.value.publishDate > Date...`) and remove the `!`